### PR TITLE
doc: control service manual

### PR DESCRIPTION
**Pull request by matzf**
Add a conceptual description of the control plane and a reference manual for the control service.

The control plane description is mostly based on the [IETF SCION control plane specification draft](https://www.ietf.org/archive/id/draft-dekater-scion-controlplane-00.html).

The reference manual for the control service describes the many command line flags, environment variables, configuration files and APIs. This documentation highlights a variety of usability shortcomings, redundancies and inconsistencies that should be addressed eventually.

- - -
This change is [Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)

### CODE REVIEW

**Review by matzf — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: all files reviewed, 2 unresolved discussions (waiting on @oncilla)

___
*[`doc/glossary.rst` line 33 at r1](https://reviewable.io/reviews/scionproto/scion/4361#-N_9gfd25yOyisf8f8el:-N_EHe93D4_lDjrH5D8e:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/47f986c86ccc7a1765ac118b7554cab0e6918571/doc/glossary.rst#L33)):*
> ```reStructuredText
>    Certificate Authority
> 
>       An entity that signs and issues digital certificates , certifying the ownership of a public
> ```

Done.
___
*[`doc/manuals/control.rst` line 14 at r1](https://reviewable.io/reviews/scionproto/scion/4361#-N_9eC_X-9gSugPasvfc:-N_E6pGb5Bah_Ze28Xsp:bcchndm) ([raw file](https://github.com/scionproto/scion/blob/47f986c86ccc7a1765ac118b7554cab0e6918571/doc/manuals/control.rst#L14)):*
<details><summary><i>Previously, oncilla (Dominik Roos) wrote…</i></summary><blockquote>

This is not accurate. A CA is not required to run as part of a core AS.

Core and CA are two separate properties of an AS.

</blockquote></details>

Done. I've added a short introduction text about the different AS roles to the overview section.

**Review by nicorusti — COMMENTED** [review]
Hi Matthias, thanks a lot, I added some minor suggestions in the sections that are derived from the IETF draft. 

I only skimmed through the rest of the changes, as I'm not too familiar with the control service. It is really great that you documented it better. When I tried to use it some time ago I would have really needed something like this :) 

> Reviewed 6 of 15 files at r1, 1 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361#-:-N_KEdYuCbDouO0qSDRr:bjqk291)* status: 13 of 15 files reviewed, 8 unresolved discussions (waiting on @matzf and @oncilla)

___
*[`doc/control-plane.rst` line 135 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K0vjJ28LitDmLL1Xb:-N_K0vjKCipRZrs3b9Ie:b-ot9snb) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L135)):*
> ```reStructuredText
> ----------
> 
> Every AS adds a signed *AS entry* to the PCBs it originates, propagates or terminates.
> ```

Here I think you mention "terminating" a PCB, however the concept of terminating a PCB is not yet introduced. If you don't know about its special meaning, it sounds a bit confusing (it almost feels like the PCB could be discarded).



_Suggestion:_
```reStructuredText
 or terminates. "Termination" consists of the case where the given AS is the first or last AS in the path segment.
```
___
*[`doc/control-plane.rst` line 137 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K3wepEOKnBHtkhAxo:-N_K3wepEOKnBHtkhAxp:b64burv) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L137)):*
> ```reStructuredText
> Every AS adds a signed *AS entry* to the PCBs it originates, propagates or terminates.
> 
> This AS entry includes the relevant network topology information for this AS-hop
> ```

In the IETF drafts, we added a lot more details about message formats (like section 2.2.1.3. for the AS entry). I'm not sure if this is too detailed for the documentation, but perhaps you could add a pointer to that section. However, the challenge today is that the text structure and section numbers in the IETF drafts might change quite a bit. Just something to keep in consideration ( either for now or for the future).
___
*[`doc/control-plane.rst` line 191 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K5Kof6YpqICEQWUdq:-N_K5Kof6YpqICEQWUdr:bczxa9x) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L191)):*
> ```reStructuredText
>   core AS that originated the PCB.
>   As a result, a core AS's path database contains all down-segments registered by the
>   non-core ASes in its customer cone.
> ```

Perhaps downstream is more appropriate? This is because the child of a child of a core AS is not necessary a customer of the core AS directly. At least in the iETF draft, we try to use "downstream" more rather than "customer" 



_Suggestion:_
```reStructuredText
downstream cone
```
___
*[`doc/control-plane.rst` line 240 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K9Jiz9z_m8mGbB_Pc:-N_K9Jiz9z_m8mGbB_Pd:b-flxugm) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L240)):*
> ```reStructuredText
> All remote path-segment lookups by the control service are cached.
> 
> On SCION end-hosts, a :doc:`SCION daemon <manuals/daemon>` is usually employed to do the
> ```

In the IETF draft we try to use endpoint (which  you also used here a few times). Here you use end-hosts. I think it would be good to make it uniform. 



_Suggestion:_
```reStructuredText
endpoint
```
___
*[`doc/control-plane.rst` line 259 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_KAN4Z7PSMwtfUeZc9:-N_KAN4Z7PSMwtfUeZcA:undefined) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L259)):*
> ```reStructuredText
> path segments to end-to-end forwarding paths.
> This path-segment combination process is done by each endpoint separately.
> Typically, end-hosts run the :doc:`SCION daemon <manuals/daemon>` which centralizes the
> ```





_Suggestion:_
```reStructuredText
endpoints
```
___
*[`doc/glossary.rst` line 63 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_KBVqd3IfMk_B9nV08:-N_KBVqd3IfMk_B9nV09:b-j1pojs) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/glossary.rst#L63)):*
> ```reStructuredText
>       An interface ID is the AS-local identifier for an inter-domain link.
> 
>       The interface ID is an arbitrary number between 1 and 65535,
> ```

IN the draft I tried to include field length when possible. As you mention that the max value is 65535, I thought we might as well indicate the field length. 



_Suggestion:_
```reStructuredText
 16 bit 
```

**Review by matzf — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: 13 of 15 files reviewed, 3 unresolved discussions (waiting on @nicorusti and @oncilla)

___
*[`doc/control-plane.rst` line 135 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K0vjJ28LitDmLL1Xb:-N_ZAklD45VfJc6IWe4V:b-icq9wi) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L135)):*
<details><summary><i>Previously, nicorusti (Nicola Rustignoli) wrote…</i></summary><blockquote>

Here I think you mention "terminating" a PCB, however the concept of terminating a PCB is not yet introduced. If you don't know about its special meaning, it sounds a bit confusing (it almost feels like the PCB could be discarded).
</blockquote></details>

True. I've  changed it to "registers", the "termination" is a part of the registration.
___
*[`doc/control-plane.rst` line 137 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K3wepEOKnBHtkhAxo:-N_bqHgT1gF49z1ue0hI:b-ckpmyn) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L137)):*
<details><summary><i>Previously, nicorusti (Nicola Rustignoli) wrote…</i></summary><blockquote>

In the IETF drafts, we added a lot more details about message formats (like section 2.2.1.3. for the AS entry). I'm not sure if this is too detailed for the documentation, but perhaps you could add a pointer to that section. However, the challenge today is that the text structure and section numbers in the IETF drafts might change quite a bit. Just something to keep in consideration ( either for now or for the future).
</blockquote></details>

Done. I've added message definitions to illustrate the AS entry and hop field which seem to be the main elements and added a reference to the "spec" draft too. The link is to a specific version of the draft, so the link will remain valid, but may not point to the latest version of the draft.
___
*[`doc/control-plane.rst` line 191 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K5Kof6YpqICEQWUdq:-N_bwFOv1U_0f_knzkqV:bjarn4z) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L191)):*
<details><summary><i>Previously, nicorusti (Nicola Rustignoli) wrote…</i></summary><blockquote>

Perhaps downstream is more appropriate? This is because the child of a child of a core AS is not necessary a customer of the core AS directly. At least in the iETF draft, we try to use "downstream" more rather than "customer" 
</blockquote></details>

Customer cone is an existing term: https://asrank.caida.org/about#customer-cone
I'll just leave it. If it's confusing, I'd rephrase to something like "the direct and indirect customer ASes".
___
*[`doc/control-plane.rst` line 240 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K9Jiz9z_m8mGbB_Pc:-N_bugPv3nDjksEvb7Kb:b-p075hr) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L240)):*
<details><summary><i>Previously, nicorusti (Nicola Rustignoli) wrote…</i></summary><blockquote>

In the IETF draft we try to use endpoint (which  you also used here a few times). Here you use end-hosts. I think it would be good to make it uniform. 
</blockquote></details>

As discussed in the glossary, endpoint vs end host: "End host is preferred where the focus is (physical or virtual) machines and the software running on them, and endpoint is used otherwise."

Here "end host" seems fine (except for the unnecessary hyphen).
___
*[`doc/control-plane.rst` line 259 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_KAN4Z7PSMwtfUeZc9:-N_bvKY02J-RSIZ4igkk:bs4q9op) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L259)):*
> ```reStructuredText
> path segments to end-to-end forwarding paths.
> This path-segment combination process is done by each endpoint separately.
> Typically, end-hosts run the :doc:`SCION daemon <manuals/daemon>` which centralizes the
> ```

Ditto.
___
*[`doc/glossary.rst` line 63 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_KBVqd3IfMk_B9nV08:-N_bvi7pDJx5E7yyEziZ:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/glossary.rst#L63)):*
<details><summary><i>Previously, nicorusti (Nicola Rustignoli) wrote…</i></summary><blockquote>

IN the draft I tried to include field length when possible. As you mention that the max value is 65535, I thought we might as well indicate the field length. 
</blockquote></details>

Done.

**Review by matzf — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: 10 of 15 files reviewed, 3 unresolved discussions (waiting on @nicorusti and @oncilla)

___
*[`doc/control-plane.rst` line 191 at r2](https://reviewable.io/reviews/scionproto/scion/4361#-N_K5Kof6YpqICEQWUdq:-N_cBeG02x8rgEszO1gF:b-eiu4ym) ([raw file](https://github.com/scionproto/scion/blob/054a4f757504fbbebb5669ddff322a6dd2596fdb/doc/control-plane.rst#L191)):*
<details><summary><i>Previously, matzf (Matthias Frei) wrote…</i></summary><blockquote>

Customer cone is an existing term: https://asrank.caida.org/about#customer-cone
I'll just leave it. If it's confusing, I'd rephrase to something like "the direct and indirect customer ASes".
</blockquote></details>

Oops, I seem to have accidentally saved with the change to "direct or indirect customer ASes". This seems also ok, I'll leave _that_ now.

**Review by oncilla — APPROVED** [review]
> Reviewed 1 of 3 files at r2, 4 of 4 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @matzf)

**Review by jiceatscion — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: all files reviewed (commit messages unreviewed), 15 unresolved discussions (waiting on @matzf)

___
*[`doc/manuals/control.rst` line 86 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagEOtV1iT4f174yLCZ:-NagEOtW14QEs0DbckpW:b-1fbgfi) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L86)):*
> ```reStructuredText
> The :program:`control` service is configured with multiple files:
> 
> - the :ref:`.toml <control-conf-toml>`,
> ```

"the .toml" -> "the .toml file"
___
*[`doc/manuals/control.rst` line 111 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagGaOHDK0QjfYutT8Q:-NagGaOHDK0QjfYutT8R:b-kx81hl) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L111)):*
> ```reStructuredText
>    .. option:: general.id = <string> (Required)
> 
>       Identifier for this control service.
> ```

An identifier



_Code quote:_
```reStructuredText
Identifier
```
___
*[`doc/manuals/control.rst` line 113 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagGkjgBVqZ1N98SkOH:-NagGkjgBVqZ1N98SkOI:bq9zub7) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L113)):*
> ```reStructuredText
>       Identifier for this control service.
> 
>       This is used to identify which parts of the :ref:`control-conf-topo` file refer to self.
> ```

are refering



_Code quote:_
```reStructuredText
refer
```
___
*[`doc/manuals/control.rst` line 129 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagHARo295QK_B73TrT:-NagHARo295QK_B73TrU:bunyttc) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L129)):*
> ```reStructuredText
>       .. Warning::
>          This should be set to ``true``, unless your service orchestration ensures that
>          failures of the dispatcher trigger a restart of :program:`control` also.
> ```

also trigger a restart of :program:`control`



_Code quote:_
```reStructuredText
trigger a restart of :program:`control` also
```
___
*[`doc/manuals/control.rst` line 150 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagHa9o94_J7MrskilX:-NagHa9o94_J7MrskilY:b32whk) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L150)):*
> ```reStructuredText
>    .. option:: api.addr = <string> (Optional)
> 
>       Address on which to expose the :ref:`control-rest-api`,
> ```

at



_Code quote:_
```reStructuredText
on
```
___
*[`doc/manuals/control.rst` line 173 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagI2Ew1iDPEWTLw_ap:-NagI2Ew1iDPEWTLw_aq:b-k06uch) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L173)):*
> ```reStructuredText
>       Local SCION address for inter-AS communication by QUIC/gRPC.
>       By default, the address specified for this control service in its ``control_service`` entry of
>       the :ref:`control-conf-topo` is used.
> ```

By default, the address used is that specified for this control service in its ``control_service`` entry of
      the :ref:`control-conf-topo`.



_Code quote:_
```reStructuredText
      By default, the address specified for this control service in its ``control_service`` entry of
      the :ref:`control-conf-topo` is used.
```
___
*[`doc/manuals/control.rst` line 221 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagJLrm7-L16CNcpgrx:-NagJLrm7-L16CNcpgry:b-io7srq) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L221)):*
> ```reStructuredText
> .. object:: ca
> 
>    .. option:: ca.mode = "disabled"|"in-process"|"delegating" (Default: "disabled")
> ```

delegating or delegated ?
___
*[`doc/manuals/control.rst` line 227 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagJG_DDSUpzbjezVQf:-NagJG_DDSUpzbjezVQg:b-8ksifc) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L227)):*
> ```reStructuredText
> 
>       If set to ``in-process``, :program:`control` handles certificate issuance requests on its own.
>       If set to ``delegated``, the certificate issuance is delegated to the service defined in
> ```

delegated or delegating ?
___
*[`doc/manuals/control.rst` line 251 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagJvOl3XGi4QeKFaHc:-NagJvOl3XGi4QeKFaHd:b32v6g) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L251)):*
> ```reStructuredText
>            based on `smallstep's step-ca <https://github.com/smallstep/certificates>`_,
>          - ask SCION vendors for proprietary CA implementations and offerings,
>          - or plug in your own CA service implementing the :file-ref:`spec/ca.gen.yaml` API.
> ```

- 



_Code quote:_
```reStructuredText
- or 
```
___
*[`doc/manuals/control.rst` line 381 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagL-iQAlBTUi77kUJK:-NagL-iQAlBTUi77kUJL:b-uwrtnr) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L381)):*
> ```reStructuredText
> -------------
> 
> The :program:`control` reads the ``control_service`` section of the :ref:`topology.json <common-conf-topo>` file.
> ```

This renders as "The **control**"
Did you mean this to read as "The program 'control'" or as "The **control** service" or as "**Control**"?
___
*[`doc/manuals/control.rst` line 401 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagM-vrDLnlNgH9RgWc:-NagM-vrDLnlNgH9RgWd:bzca8e3) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L401)):*
> ```reStructuredText
> Propagation
>    Propagation is the process of receiving a beacon from a neighbor AS, extending it
>    with the own AS entry and forwarding it to downstream neighbor ASes.
> ```

the own -> "the local"? "one's own"?
___
*[`doc/manuals/control.rst` line 470 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagN20S0tJoHfU3tlmS:-NagN20S0tJoHfU3tlmT:b-66b1qx) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L470)):*
> ```reStructuredText
>    .. warning::
> 
>       Keep this parameter reasonably low to avoid explosion of beacon numbers.
> ```

an explosion



_Code quote:_
```reStructuredText
explosion
```
___
*[`doc/manuals/control.rst` line 518 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagNoL69l_GgYQy-Pju:-NagNoL69l_GgYQy-Pjv:b-h412cz) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L518)):*
> ```reStructuredText
>    Filters are applied when a beacon is received, resulting in a "usage" classification of the
>    beacon that is stored in the local beacon database.
>    When the policy is changed, it will thus only be effective for newly received beacons.
> ```

Therefore, when the policy is changed, it will



_Code quote:_
```reStructuredText
When the policy is changed, it will thus 
```
___
*[`doc/manuals/control.rst` line 526 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagNf0VEAww6by-cOra:-NagNf0W54XlHF2yV1Dp:b-66mqkl) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L526)):*
> ```reStructuredText
> 
>       There are plans to extend this functionality but so far there are no concrete proposals.
>       If you're interested to work on this, get in contact in the :ref:`chat <slack>` or create a
> ```

in working



_Code quote:_
```reStructuredText
to work
```
___
*[`doc/manuals/control.rst` line 648 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagP5zO6t9dESB_-U2_:-NagP5zO6t9dESB_-U2a:b-uf0jy) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L648)):*
> ```reStructuredText
> If the configuration file exists, it must be syntactically valid..
> 
> The structure of the configuration is presented as a pseudo-JSON with a more detailed explanation
> ```

pseudo-JSON



_Code quote:_
```reStructuredText
a pseudo-JSON
```

**Review by matzf — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: all files reviewed (commit messages unreviewed), 15 unresolved discussions (waiting on @jiceatscion)

___
*[`doc/manuals/control.rst` line 86 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagEOtV1iT4f174yLCZ:-NavftmY-fC8_RvEruHo:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L86)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

"the .toml" -> "the .toml file"
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 111 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagGaOHDK0QjfYutT8Q:-Navfv7-2VURl3XCiXrS:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L111)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

An identifier
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 113 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagGkjgBVqZ1N98SkOH:-Navfvra-wb_leMGurYS:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L113)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

are refering
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 129 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagHARo295QK_B73TrT:-NavfwVP3UGM5ack17Ld:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L129)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

also trigger a restart of :program:`control`
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 150 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagHa9o94_J7MrskilX:-NavfwypA3opY4LICq7j:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L150)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

at
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 173 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagI2Ew1iDPEWTLw_ap:-Navfxxo9UEHzVnnPnSN:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L173)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

By default, the address used is that specified for this control service in its ``control_service`` entry of
      the :ref:`control-conf-topo`.
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 221 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagJLrm7-L16CNcpgrx:-NavfyhG-Ig_Nrkq2yJF:b-rqvsi3) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L221)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

delegating or delegated ?
</blockquote></details>

Done, "delegating".
___
*[`doc/manuals/control.rst` line 227 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagJG_DDSUpzbjezVQf:-Navg0HR8UnXbd8fxJFa:b-rqvsi3) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L227)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

delegated or delegating ?
</blockquote></details>

Done, "delegating".
___
*[`doc/manuals/control.rst` line 251 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagJvOl3XGi4QeKFaHc:-NavgVt7EIuSlz9cG7jU:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L251)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

- 
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 381 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagL-iQAlBTUi77kUJK:-NavhQ-L1axLjBnKZTb6:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L381)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

This renders as "The **control**"
Did you mean this to read as "The program 'control'" or as "The **control** service" or as "**Control**"?
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 401 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagM-vrDLnlNgH9RgWc:-Navgk1HBlVDZq2yMQ7x:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L401)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

the own -> "the local"? "one's own"?
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 470 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagN20S0tJoHfU3tlmS:-Navgpn8-bZfRiIT8ySp:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L470)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

an explosion
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 518 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagNoL69l_GgYQy-Pju:-NavgvXc-4jtRuEoGFH-:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L518)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

Therefore, when the policy is changed, it will
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 526 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagNf0VEAww6by-cOra:-Navgy3V1Ub9z6tWuwtC:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L526)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

in working
</blockquote></details>

Done.
___
*[`doc/manuals/control.rst` line 648 at r4](https://reviewable.io/reviews/scionproto/scion/4361#-NagP5zO6t9dESB_-U2_:-Navh5lk3Yd9wPlqsu5J:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/58232c10c43e0318f7d5456a13e19bbfdb0c06b0/doc/manuals/control.rst#L648)):*
<details><summary><i>Previously, jiceatscion wrote…</i></summary><blockquote>

pseudo-JSON
</blockquote></details>

Done.

**Review by jiceatscion — COMMENTED** [review]
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4361)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @matzf)
